src/main/java/org/opentripplanner/hasura_client/VehiclePositionsGetter.java

```diff
@@ -9,11 +9,15 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Collections;
+import java.util.Objects;
 import java.util.Set;
 
 public class VehiclePositionsGetter extends HasuraGetter<VehicleDescription, Vehicle> {
     private static final Logger LOG = LoggerFactory.getLogger(VehiclePositionsGetter.class);
 
+    private VehiclePositionsMapper mapper;
```

Comment:
Those are only style changes, right?

Reply:
I have compared it to the dev-1.x branch locally and the only difference was the removal of an empty line above the "private static final Logger LOG ..." initialization